modules/hdf4_handler/HDF4Module.cc

```diff
@@ -42,12 +41,13 @@
 #include <BESDebug.h>
 
 using namespace std;
+const string HDF4_CATALOG="catalog";
```

Comment:
In this case I'm not so sure the change is an improvement. We have had mixed results with 'string' objects in the global namespace. This should be OK, but be wary about advice to do this with class members. SonarScan does not seem to like the preprocessor. I don't agree with it on that point. However, if this works and does not cause an issue during program exit, it's fine.

Reply:
I agree with you. I won't change any MACRO like this in the future.